demo/main.js

```diff
@@ -1484,7 +1484,7 @@ function onDemoConfigChanged(firstLoad) {
   )}&demoConfig=${serializedDemoConfig}`;
 
   $('#StreamPermalink').html(`<a href="${permalinkURL}">${permalinkURL}</a>`);
-  if (!firstLoad && window.location !== permalinkURL) {
+  if (!firstLoad && window.location.toString() !== permalinkURL) {
```

Comment:
did it not not work before? `window.location` when used as a string should have worked 🤔 It's an alias of `window.location.href`

Reply:
ah nevermind I'm being stupid

```suggestion
  if (!firstLoad && window.location.href !== permalinkURL) {
```

would be better though as that's the actual property